golang/TODO.md

```diff
@@ -0,0 +1,105 @@
+h1. Functionality
+
+[ ] Cancel outbound requests
```

Comment:
This doesn't render properly in github. If you want it to, I think you need to make this a list, e.g. prefix with `- `